.github/workflows/pypi_release.yml

```diff
@@ -0,0 +1,27 @@
+name: PyPI Release
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  test_pypi_release:
```

Comment:
Should this  job be called `pypi_release?`

Reply:
Yes, thank you for the fix